src/styleRe.re

```diff
@@ -15,9 +14,8 @@ type style =
       string [ | `value AnimatedRe.Value.t | `interpolation AnimatedRe.Interpolation.t];
 
 let combine a b => {
```

Comment:
Does the array syntax also work with `StyleSheets`, i.e. can a value be an array? Otherwise we would need two separate functions